hil/api.py

```diff
@@ -1227,6 +1227,16 @@ def list_headnode_images():
     return json.dumps(valid_imgs)
 
 
+# Extension code #
+#################
+@rest_call('GET', '/active_extensions', Schema({}))
```

Comment:
Since I'm not expecting any input, should I just validate that nothing is there?